mvpa2/tests/test_mapper.py

```diff
@@ -457,3 +457,15 @@ def test_addaxis():
     # add multiple axes
     ds4 = AddAxisMapper(pos=4)(ds)
     assert_array_equal(ds4.shape, ds.shape + (1, 1))
+
+def test_identity_mapper():
+    idm = IdentityMapper()
+    # doesn't matter what you throw at it
+    for s in ('i_am_the_test',
+              range(5),
+              np.arange(12).reshape(4,3),
+              Dataset(np.arange(12).reshape(4,3))):
```

Comment:
you could have used ```@sweep_args```, which would be handy if e.g. only one of them fails -- all the others still get tested etc